HTTP2-Core.package/SqueakSSL.extension/instance/alpnProtos..st

```diff
@@ -0,0 +1,3 @@
+*HTTP2-Core
+alpnProtos: aProtocolSet
```

Comment:
Bitte Abkürzungen vermeiden - alpnProtocols:

Reply:
Die Abkürzungen entsprechen 1:1 den Abkürzungen, die auch in der OpenSSL-API genutzt werden und sind daher aus Konsistenzgründen so gewählt.

Reply:
Ist das nicht unnötig? Es gibt übrigens in SqueakSSL auch Getter für certName etc.

Reply:
naja, meinetwegen